OneCommunityGlobal/HighestGoodNetworkApp
# Roberto fix removing user from profile

# Description
When trying to remove a user from a team within's the user's profile page it doesn't update the database only state. Additionally when you go to a team's member list and either add or remove a user from there then visit that user's profile to see the update it will have a very delayed response.

<img width="820" alt="Screenshot 2023-09-28 at 5 14 05 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/117053202/6af0b7f0-cd38-420e-a22f-c644bbcd27b0">
<img width="823" alt="Screenshot 2023-09-28 at 5 13 57 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/117053202/dd52eb5a-1d50-4fa1-acfd-fd74bd3572b7">

## Related PRS (if any):
This frontend PR is related to the [#550](https://github.com/OneCommunityGlobal/HGNRest/pull/550) backend PR.
…

## Main changes explained:
- Imported and used deleteTeamMember function to TeamsTab.jsx to ping the backend and update the database.
- Added a boolean variable to UserProfile.jsx to indicate when the save button is clicked to execute the function
…

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as admin/owner user
4. go to dashboard→ Other Links→ Teams→ Select A Team→ Members→ Add a User to Team→ Visit that user's profile
5. verify that the user's profile is updated immediately after a refresh
6. User's profile→ Teams section→ Remove from Team→ Save Changes→ Visit that Team
7. verify that removing the user from a team from within a user's profile actually updates the team member list
8. repeat step 4 and 5 + also remove a user from team member list and make sure it updates user profile immediately

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/117053202/7f044ee4-62ed-434d-bea0-46801f1565de



## Note:
Adding user to a Team from user's profile page won't actually update the Team's member list
